Client.cpp

```diff
@@ -218,7 +218,65 @@ void Client::whois(const std::vector<std::string> &msg) {
 }
 
 void Client::who(const std::vector<std::string> &msg) { (void)msg; }
-void Client::privmsg(const std::vector<std::string> &msg) { (void)msg; }
+
+void Client::privmsg(const std::vector<std::string> &msg) {
+  if (msg.size() < 2) {
+    createMessage(Server::ERR_NORECIPIENT, msg[0]);
+    return;
+  }
+  if (msg.size() < 3) {
+    createMessage(Server::ERR_NOTEXTTOSEND, msg[0]);
+    return;
+  }
+  if (msg[1].size() > 0 && (std::string(CHANNEL_PREFIXES).find(msg[1][0]) != std::string::npos)) {
+    _messageChannel(msg);
+  } else {
+    _messageClient(msg);
+  }
+
+}
+
+void Client::_messageClient(const std::vector<std::string> &msg) {
+  std::string target = msg[1];
+  if (target[0] == ':') {
+    target = target.substr(1);
+  }
+  std::string text = msg[2];
+  if (text[0] == ':') {
+    text = text.substr(1);
+  }
```

Comment:
```suggestion
  const std::string &target = msg[1];
  const std::string &text = msg[2];
```
  I think this is not needed since #46. I remove the colon in the parse().